Makefile

```diff
@@ -31,7 +31,7 @@
 include config/version
 
 SHELL = /bin/sh
-SYSTEM ?= $(shell config/config.guess | cut -d - -f 3 | sed -e 's/[0-9\.]//g;')
+SYSTEM ?= $(shell config/config.guess | cut -d - -f 3 | sed -e 's/\.//g' -e 's/[0-9]\{1,\}.*//')
```

Comment:
> I wasn't sure if periods can appear anywhere other than in the release number so I retained the previous behavior of removing all periods regardless where they are.

If periods can only appear within the release number, then this can be simplified with basic REs to:

```suggestion
SYSTEM ?= $(shell config/config.guess | cut -d - -f 3 | sed -e 's/[0-9\.]\{1,\}.*//')
```

or with extended REs to:

```suggestion
SYSTEM ?= $(shell config/config.guess | cut -d - -f 3 | sed -E -e 's/[0-9\.]+.*//')
```

Old versions of GNU sed don't know the standard `-E` flag for enabling extended REs while old versions of BSD sed (and all versions of BSD sed bundled with macOS) don't know GNU sed's `-r` flag for enabling extended REs. This inconsistency could be a reason to continue to prefer basic REs.